lib/src/components/month_view_components.dart

```diff
@@ -166,7 +161,6 @@ class FilledCell<T> extends StatelessWidget {
                           padding: const EdgeInsets.all(2.0),
                           alignment: Alignment.center,
                           child: Row(
-                            mainAxisSize: MainAxisSize.max,
```

Comment:
Why we removed `MainAxisSize`?

Reply:
Values we provided as arguments are default values for Column. So whether we provide it or not doesn't matter.